core/types/transaction.go

```diff
@@ -432,6 +432,24 @@ func TxDifference(a, b Transactions) Transactions {
 	return keep
 }
 
+// TxDifferenceHash returns a new set which is the difference between a and b.
+func TxDifferenceHash(a, b []common.Hash) []common.Hash {
```

Comment:
```suggestion
func HashDifference(a, b []common.Hash) []common.Hash {
```
Nothing specifically tx-related with this method